pages/development/processes/pull-request.mdx

```diff
@@ -0,0 +1,114 @@
+# Pull Request conventions
+
+## Branch name
+```
+<Task ID>-<Task name>
+```
+- Task ID: is the Task ID on Redmine or the other task management tool
+- Task name: is the name of the task in KebabCase
+
+#### Examples
+- The Redmine task: `#2131 [FE] Implement api add device`
+  
+  The branch name: `2131-implement-api-add-device`
```

Comment:
@haihuynhDF 
Do we need to have the prefix for branch name, Hai?

https://medium.com/@abhay.pixolo/naming-conventions-for-git-branches-a-cheatsheet-8549feca2534

Reply:
Yes, it's what I am looking
Let me update